publishable/lang/tr/voyager.php

```diff
@@ -267,31 +281,61 @@
         'model_name'                => 'Model Adı',
         'model_name_ph'             => 'ex. \App\User, Eğer boş ise tablo adını deneyin',
         'name_warning'              => 'Lütfen indeks eklemden önce kolon adı belirleyin',
-        'no_composites_warning'     => 'This table has composite indexes. Please note that they are not supported '.
-            'at the moment. Be careful when trying to add/remove indexes.',
-        'null'                  => 'Boş',
-        'optional_details'      => 'İsteğe Bağlı Ayrıntılar',
-        'primary'               => 'BİRİNCİL',
-        'server_pagination'     => 'Sunucu-taraflı sayfalama',
-        'success_create_table'  => 'Başarılı tablo oluşturuldu :table ',
-        'success_created_bread' => 'Başarılı yeni BREAD oluşturuldu',
-        'success_delete_table'  => 'Başarılı tablo silindi :table table',
-        'success_remove_bread'  => 'Başarılı silindi BREAD şurdan :datatype',
-        'success_update_bread'  => 'Başarılı güncellendi :datatype BREAD',
-        'success_update_table'  => 'Başarılı tablo güncellendi :table table',
-        'table_actions'         => 'Tablo işlemleri',
-        'table_columns'         => 'Tablo kolanları',
-        'table_has_index'       => 'Tablo zaten birincil anahtarı var.',
-        'table_name'            => 'Tablo Adı',
-        'table_no_columns'      => 'Tabloda hiç kolan bulunamadı...',
-        'type'                  => 'Tip',
-        'type_not_supported'    => 'Bu tip desteklenöiyor',
-        'unique'                => 'BENZERSİZ',
-        'unknown_type'          => 'Bilinmeyen Tip',
-        'update_table'          => 'Tabloyu güncelle',
-        'url_slug'              => 'Link yazısı (benzersiz olmalı)',
-        'url_slug_ph'           => 'Link yazısı (ex. gonderi)',
-        'visibility'            => 'Görünür',
+        'no_composites_warning'     => 'This table has composite indexes. Please note that they are not supported '.'at the moment. Be careful when trying to add/remove indexes.',
```

Comment:
This line is still in English. Did you intend to translate it?

Reply:
Oh sorry, I really didn't notice it. Translating it now.